builder/vmware/iso/step_create_vmx.go

```diff
@@ -51,6 +51,8 @@ type vmxTemplateData struct {
 	Parallel_Bidirectional string
 	Parallel_Filename      string
 	Parallel_Auto          string
+
+	HardwareAssistedVirtualization string
```

Comment:
Oops I forgot to include the type change. This should fix the failures. 
```suggestion
	HardwareAssistedVirtualization bool
```